bin/Powheg/production/2017/13TeV/Higgs/MSSM/ggA_MuMu_NNPDF31_13TeV/powheg.input_template

```diff
@@ -0,0 +1,99 @@
+!!!!!!!!!   parameters that define the process !!!!!!!!!!!!!!!!!!
+higgstype 3 ! higgs type: 1 = light CP even; 2 = heavy CP even; 3 = pseudoscalar
+ih1   1           ! hadron 1 (1 for protons, -1 for antiprotons)
+ih2   1           ! hadron 2 (1 for protons, -1 for antiprotons)
+ebeam1 6500     ! energy of beam 1
+ebeam2 6500    ! energy of beam 2
+hdecaymode 8      ! PDG code for first decay product of the higgs
+! allowed values are:  0 all decay channels open
+!	               1-6 d dbar, u ubar,..., t tbar (as in HERWIG)
+!	               7-9 e+ e-, mu+ mu-, tau+ tau-
+!		       10  W+W-
+!	               11  ZZ
+!	               12  gamma gamma
+hwidth 10d0	
+hdecaywidth 0	! 1 read Higgs decay width from HDECAY file
+zerowidth 0 ! Control if the Higgs boson is to be produced on-shell or not: 1 = On-Shell; 0 = Off-shell with Breit-Wigner
+bwshape 1	! BW shape: 1 running width BW, 2 fixed width BW
+masswindow_high 5d0 ! number of widths around hmass in the BW for an off-shell Higgs boson  upper limit
+masswindow_low 5d0  ! number of widths around hmass in the BW for an off-shell Higgs boson  lower limit
+ew 0                     ! ew = 0 disable EW corrections - ew = 1 enable EW corrections
+fastew 1                   ! fast ew corrections evaluations by mass sampling
```

Comment:
why is this 1 while ew is 0 ? maybe not relevant in this case?

Reply:
Yes, it is not relevant in this case